experimental/skills/DialogToDialog/DialogRootBot/Dialogs/MainDialog.cs

```diff
@@ -0,0 +1,230 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Generic;
+using System.Net.Http;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.Bot.Builder;
+using Microsoft.Bot.Builder.Dialogs;
+using Microsoft.Bot.Builder.Integration.AspNet.Core.Skills;
+using Microsoft.Bot.Connector.Authentication;
+using Microsoft.Bot.Schema;
+using Microsoft.Extensions.Configuration;
+using Newtonsoft.Json;
+
+namespace Microsoft.BotBuilderSamples.DialogRootBot.Dialogs
+{
+    /// <summary>
+    /// A sample MainDialog that uses SkillDialog to call skills.
+    /// </summary>
+    public class MainDialog : ComponentDialog
+    {
+        // Note: in this example we using only one skill.
```

Comment:
either
```suggestion
        // Note: this example uses only one skill.
```
or
```suggestion
        // Note: in this example we are using only one skill.
```

Reply:
Updated in source

---
In reply to: [356214270](https://github.com/microsoft/BotBuilder-Samples/pull/2032#discussion_r356214270) [](ancestors = 356214270)